core/src/main/scala/cats/data/IorT.scala

```diff
@@ -435,6 +435,12 @@ private[data] abstract class IorTInstances extends IorTInstances1 {
       Monad[IorT[M, E, ?]]
     }
   }
+
+  implicit def catsDataDeferForIor[F[_], E](implicit F: Defer[F]): Defer[IorT[F, E, ?]] =
```

Comment:
missing `T` after `catsDataDeferForIor`?

Reply:
looks like we are still missing the T.

Reply:
I got you. https://github.com/typelevel/cats/pull/2688/files